CHANGELOG.txt

```diff
@@ -1,3 +1,6 @@
+4.0.1
+* Fix JavaDoc errors
```

Comment:
We also added an extra step for verifying the docs generation in CI, but I didn't include it here since it seems we were not adding CI related changes in the `CHANGELOG.txt` file.